### ISSUE TITLE
Configuration Options for SSL

### ISSUE THREAD

**Issue by JustOneMoreBlock** [Block2Block/HubParkour#10]
Add configuration options for SSL.

The `MySQL` tables failed to create to-do SSL not being enabled and most shared hosts don't have this.

I have found a workaround for this temporarily.

On the `Database` line. You can add the following URL since you use jdbc.
```
        Database: "parkour?verifyServerCertificate=false&useSSL=false&requireSSL=false"
```

Errors
```
[01:54:44] [Server thread/WARN]: 	... 17 more
[01:54:44] [Server thread/WARN]: Caused by: javax.net.ssl.SSLHandshakeException: No appropriate protocol (protocol is disabled or cipher suites are inappropriate)
[01:54:44] [Server thread/WARN]: 	at sun.security.ssl.HandshakeContext.<init>(HandshakeContext.java:171)
[01:54:44] [Server thread/WARN]: 	at sun.security.ssl.ClientHandshakeContext.<init>(ClientHandshakeContext.java:98)
[01:54:44] [Server thread/WARN]: 	at sun.security.ssl.TransportContext.kickstart(TransportContext.java:220)
[01:54:44] [Server thread/WARN]: 	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:428)
[01:54:44] [Server thread/WARN]: 	at com.mysql.jdbc.ExportControlled.transformSocketToSSLSocket(ExportControlled.java:186)
[01:54:44] [Server thread/WARN]: 	... 36 more
[01:54:44] [Server thread/INFO]: [HubParkour] Plugin successfully enabled!
[01:54:44] [Server thread/INFO]: [HubParkour] Your HubParkour version is up to date!
[01:54:44] [Server thread/INFO]: Done (1.225s)! For help, type "help"
[01:54:44] [Server thread/INFO]: Timings Reset
```

Tested on 2.5.5

**Comment by Block2Block** [comment]
This will be addressed in the 2.6 version

**Comment by Block2Block** [comment]
Issue fixed in latest version
